### PR TITLE
update tuner to use version.py

### PR DESCRIPTION
**Pull request by SinaChavoshi**

### CODE REVIEW

File: src/python/requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+tensorflow-datasets
```
Comment (pavithrasv) [inline src/python/requirements.txt:1]: Why is this change required?

Comment (SinaChavoshi) [reply src/python/requirements.txt:1]: Removing it, Good catch that got picked up accidentally.
